Examples/ObjectiveC/LabelPlacementExample.m

```diff
@@ -0,0 +1,40 @@
+#import "LabelPlacementExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleLabelPlacement = @"LabelPlacementExample";
+
+@interface LabelPlacementExample ()<MGLMapViewDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation LabelPlacementExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.mapView = [[MGLMapView alloc] initWithFrame: CGRectMake(0, 0, self.view.bounds.size.width, self.view.bounds.size.height)];
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    
+    // Set the map's initial style, center coordinate, and zoom level
+    self.mapView.styleURL = [MGLStyle streetsStyleURL];
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(37.791282, -122.396301)
+                            zoomLevel:15.0
+                             animated:NO];
+    self.mapView.delegate = self;
+    [self.view addSubview:self.mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    MGLSymbolStyleLayer *poiLabelLayers = (MGLSymbolStyleLayer *)[self.mapView.style layerWithIdentifier:@"poi-label"];
+    poiLabelLayers.textAnchor = nil;
+    poiLabelLayers.textOffset = nil;
+    poiLabelLayers.symbolPlacement = nil;
+    poiLabelLayers.textOffset = nil;
+//    poiLabelLayers.textVariableAnchor = [NSExpression expressionForConstantValue:@[@(MGLTextAnchorTop), @(MGLTextAnchorRight)]];
```

Comment:
If you intend to keep these comments in, can you explain what uncommenting them will do?

Reply:
This is the code that does the label placement. But I don't have a beta release that contains such code, that's why I commented this code while waiting for beta.2. Once that is released will uncomment.